frontend/src/features/dsPreview/DatasetPreviewPage.tsx

```diff
@@ -97,12 +103,12 @@ const DatasetPreviewPage: React.FC<DatasetPreviewPageProps> = ({
                     </div>
                     {/* Bottom of the box */}
 
-                    {dataset.meta && dataset.meta.description && (
-                      <div className='pt-4 text-gray-400 text-xs tracking-wider mb-2'>{dataset.meta.description}</div>
-                    )}
+
+                    <div className='pt-4 text-gray-400 text-xs tracking-wider mb-2'>{(dataset.meta && dataset.meta.description) || 'No Description'}</div>
```

Comment:
use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

```suggestion
                    <div className='pt-4 text-gray-400 text-xs tracking-wider mb-2'>{ dataset.meta?.description  || 'No Description'}</div>
```